mutiny-core/src/nodemanager.rs

```diff
@@ -556,6 +557,26 @@ impl<S: MutinyStorage> NodeManager<S> {
 
         let logger = Arc::new(MutinyLogger::with_writer(stop.clone(), storage.clone()));
 
+        // Need to prevent other devices from running at the same time
+        if !c.skip_device_lock {
+            storage.set_device_lock()?;
+        }
+
+        let storage_clone = storage.clone();
+        let logger_clone = logger.clone();
+        let stop_clone = stop.clone();
+        utils::spawn(async move {
+            loop {
+                if stop_clone.load(Ordering::Relaxed) {
+                    break;
+                }
+                sleep((DEVICE_LOCK_INTERVAL_SECS * 1_000) as i32).await;
```

Comment:
Each second seems like too much, if we're doing 2m limits then we can probably just do 30s here.

Reply:
this is every minute, `DEVICE_LOCK_INTERVAL_SECS` is 60 and time 1k to go from seconds to milliseconds